instagram/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:17-alpine
+
+WORKDIR /usr/app/instagram
+
```

Comment:
You still need to include the previous files you had here so that we can build the development environment. So the `package.json` etc needs to be included here. Right now, this container doesn't add much value.